### PR TITLE
added --verbose option, which forwards stdout/err

### PR DESCRIPTION
**Pull request by scott113341**
#21

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -76,46 +79,58 @@ function getCurrentBranch() {
...
 
-function resetBranch(branch, detach) {
-  var detached = detach ? '--detach ' : '';
+function resetBranch(branch, detach, verbose) {
+  var detached = detach ? '--detach' : null;
   return execCmd(
-    'git checkout -f ' + detached + branch,
-    'problem resetting branch'
+    'git',
+    ['checkout', '-f', detached, branch].filter(function(a) { return a; }),
```
Comment (rtsao) [inline index.js]: Nice

File: index.js [hunk 2]
```diff
@@ -157,17 +174,32 @@ function handleError(err) {
   process.exit(1);
 }
 
-function execCmd(cmd, errMessage) {
+function execCmd(cmd, args, errMessage, verbose) {
   return new Promise(function(resolve, reject) {
-    exec(cmd, function(error, stdout, stderr) {
-      error ? reject(errMessage) : resolve(stdout);
+    verbose ? console.log(cmd, args.join(' ')) : null;
```
Comment (rtsao) [inline index.js:179]: Nice 👍

File: index.js (hunk 1 above)
Comment (rtsao) [inline index.js]:
```js
['checkout', '-f', detached, branch].filter(Boolean),
```

File: index.js (hunk 2 above)
Comment (scott113341) [reply index.js:179]: Perfectly leguel

File: index.js (hunk 1 above)
Comment (scott113341) [reply index.js]: Swag, I've never seen or thought of that before

**Review by rtsao — APPROVED** [review]

**Pull request merged**